contracts/bonding/IBondingManager.sol

```diff
@@ -20,7 +20,7 @@ contract IBondingManager {
     function setActiveTranscoders() external;
     function updateTranscoderWithFees(address _transcoder, uint256 _fees, uint256 _round) external;
     function slashTranscoder(address _transcoder, address _finder, uint256 _slashAmount, uint256 _finderFee) external;
-    function electActiveTranscoder(uint256 _maxPricePerSegment, uint256 _block, uint256 _round) external view returns (address);
+    function electActiveTranscoder(uint256 _maxPricePerSegment, bytes32 _rand, uint256 _round) external view returns (address);
```

Comment:
same comment about `_rand`, as it's not obvious what that is supposed to represent.